test-clients/src/test/java/E2EPackageRunner.java

```diff
@@ -21,136 +21,34 @@
 import com.hedera.services.bdd.spec.HapiApiSpec;
 import com.hedera.services.bdd.spec.props.JutilPropertySource;
 import com.hedera.services.bdd.suites.HapiApiSuite;
-import com.hedera.services.bdd.suites.autorenew.AccountAutoRenewalSuite;
-import com.hedera.services.bdd.suites.autorenew.AutoRemovalCasesSuite;
-import com.hedera.services.bdd.suites.autorenew.GracePeriodRestrictionsSuite;
-import com.hedera.services.bdd.suites.autorenew.MacroFeesChargedSanityCheckSuite;
-import com.hedera.services.bdd.suites.autorenew.NoGprIfNoAutoRenewSuite;
-import com.hedera.services.bdd.suites.autorenew.TopicAutoRenewalSuite;
+import com.hedera.services.bdd.suites.autorenew.*;
 import com.hedera.services.bdd.suites.compose.LocalNetworkCheck;
 import com.hedera.services.bdd.suites.compose.PerpetualLocalCalls;
-import com.hedera.services.bdd.suites.consensus.AssortedHcsOps;
-import com.hedera.services.bdd.suites.consensus.ChunkingSuite;
-import com.hedera.services.bdd.suites.consensus.SubmitMessageSuite;
-import com.hedera.services.bdd.suites.consensus.TopicCreateSuite;
-import com.hedera.services.bdd.suites.consensus.TopicDeleteSuite;
-import com.hedera.services.bdd.suites.consensus.TopicGetInfoSuite;
-import com.hedera.services.bdd.suites.consensus.TopicUpdateSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractCallLocalSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractCallSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractCreateSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractDeleteSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractGetBytecodeSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractGetInfoSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractMusicalChairsSuite;
-import com.hedera.services.bdd.suites.contract.hapi.ContractUpdateSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.BalanceOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.CallCodeOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.CallOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.CreateOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.DelegateCallOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.ExtCodeCopyOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.ExtCodeHashOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.ExtCodeSizeOperationSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.GlobalPropertiesSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.SStoreSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.SelfDestructSuite;
-import com.hedera.services.bdd.suites.contract.opcodes.StaticCallOperationSuite;
+import com.hedera.services.bdd.suites.consensus.*;
+import com.hedera.services.bdd.suites.contract.hapi.*;
+import com.hedera.services.bdd.suites.contract.opcodes.*;
```

Comment:
Hedera best practices guidelines specify we should avoid importing with '*'. 
You can change the IntelliJ settings not to combine imports when auto-formatting the files:
Editor -> Code Style -> Java -> Imports tab -> set `Class count...` and `Names count.....` properties to 99

Reply:
Also, there are no other changes in this file...